custom/spritereplacements.txt

```diff
@@ -59,7 +59,7 @@ Grandma.bin,Grandma,female,true,7
 Katarin.bin,Katarin,female,true,0
 Link.bin,Link,male,true,1,link-portrait.bin
 Mini.bin,Mini,neutral,true
-MrSaturn.bin,Mr Saturn,male,true,,mrsaturn-portrait.bin
+MrSaturn.bin,Mr Saturn,male,true,,mr-saturn-portrait.bin
```

Comment:
you know I think that canonically there are female Mr Saturns

Reply:
Are there? Feel free to change it to neutral then.